### ISSUE TITLE
Can't print non-ascii text to windows terminal

### ISSUE THREAD

**Issue by SepehrRasouli** [gleitz/howdoi#456]
howdoi can't print non-ASCII text to windows terminal (persian text in this case)
* I used colorama feature in this screenshot :
![Screenshot 2022-03-01 175310](https://user-images.githubusercontent.com/81516241/156186537-ef528016-288f-47b9-89a1-9d451fce9931.jpg)

* This one is without colorama :
![Screenshot 2022-03-01 175334](https://user-images.githubusercontent.com/81516241/156186776-2ecd631f-80ec-4d1b-954f-1a080dedf1af.jpg)

**Comment by gleitz** [comment]
Huh, so with or without colorama we still have the issue if there is any non-ASCII text.

Any in both cases, it also breaks the colorization (since we see characters like `[49m`

**Comment by SepehrRasouli** [comment]
Yes , but the colors work if there aren't any non-ascii chars in the output.
